services/gafaelfawr/values-bleed.yaml

```diff
@@ -1,5 +1,8 @@
 gafaelfawr:
   host: bleed.lsst.codes
+  image:
+    tag: "tickets-DM-26078"
```

Comment:
So this is just for testing this version on bleed, and will go back to master later?

Reply:
Yes, indeed. As soon as the next Gafaelfawr is released (probably with the OpenID Connect support I'm working on now).